File: src/test/java/test/WolframTest.java
package test;

import avalon.tool.WolframGetter;
import org.jdom2.JDOMException;
import org.jdom2.input.SAXBuilder;

import java.io.File;
import java.io.IOException;
import java.util.List;

/**
 * Created by Eldath Ray on 2017/6/11 0011.
 *
 * @author Eldath Ray
 */
public class WolframTest {
    public static void main(String[] args) throws IOException, JDOMException {
        SAXBuilder builder = new SAXBuilder();
        List<WolframGetter.WolframPod> pods = WolframGetter.get(builder.build(
                new File("F:/query.xml")).getRootElement());
        for (WolframGetter.WolframPod thisPod : pods) {
            System.out.println(thisPod.getTitle());
            System.out.println(thisPod.getPlaintext());
            System.out.print("---");
        }
    }
}

File: src/test/java/test/WolframTest.scala
package test

import java.io.{File, IOException}

import avalon.extend.WolframXMLParser
import org.jdom2.JDOMException
import org.jdom2.input.SAXBuilder

/**
  * Created by Eldath Ray on 2017/6/11 0011.
  *
  * @author Eldath Ray
  */
object WolframTest {
  @throws[IOException]
  @throws[JDOMException]
  def main(args: Array[String]): Unit = {
    val builder = new SAXBuilder
	  val pods = WolframXMLParser.get(builder.build(new File("F:/query.xml")).getRootElement)
    for (thisPod <- pods) {
	    System.out.println(thisPod.title)
	    System.out.println(thisPod.plaintext)
      System.out.print("---")
    }
  }
}